.github/workflows/release.yml

```
@@ -19,694 +19,436 @@ permissions:
+          name: dash-evo-tool-macos-x86_64
+          path: artifacts/macos-x86_64/
+
+      - name: Download Windows artifacts
         uses: actions/download-artifact@v4
         with:
-          name: dash-evo-tool-windows.zip
+          name: dash-evo-tool-windows-x86_64
+          path: artifacts/windows/
 
       - name: Publish release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)